### PR TITLE
Linting and Formatting chapters update

### PR DESCRIPTION
**Pull request by sapegin**
* Prettier supports TypeScripts
* Prettier supports CSS
* Prettier has format files
* Stylelint has --fix mode and can format code without stylefmt

### CODE REVIEW

File: manuscript/code-quality/01-linting.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ All ES6 features are supported and [babel-eslint](https://www.npmjs.com/package/
 
 ESLint has many plugins, for example:
 
-* [eslint-plugin-compat](https://www.npmjs.com/package/eslint-plugin-compat) — checks browser compatibility using `browserslist` file, [caniuse](http://caniuse.com/) and [@kangax’s compat](http://kangax.github.io/compat-table/es6/) table.
+* [eslint-plugin-compat](https://www.npmjs.com/package/eslint-plugin-compat) — checks browser compatibility using *browserslist* file, [caniuse](http://caniuse.com/) and [@kangax’s compat](http://kangax.github.io/compat-table/es6/) table.
```
Comment (bebraw) [inline manuscript/code-quality/01-linting.md]: It's `.browserslistrc` now. They changed the name. I guess we should just link to [browserslist npm](https://www.npmjs.com/package/browserslist) here, though.

File: manuscript/code-quality/01-linting.md [hunk 2]
```diff
@@ -226,14 +226,14 @@ Update your *package.json* like this:
 
 This configuration will:
 
-* Every time you commit a `.js` file:
+* Every time you commit a *.js* file:
 
   1. Run ESLint with autofixing on files you are committing.
   2. Run Jest tests related to files you are committing.
-  3. Format files you are committing with Prettier. The idea is discussed in detail in the next chapter.
+  3. Format files you are committing with Prettier. (See the next chapter for more details.)
```
Comment (bebraw) [inline manuscript/code-quality/01-linting.md]: We should replace "the next chapter" with `*<chapter name>*`. Then the site can pick that up and generate a link.

File: manuscript/code-quality/01-linting.md [hunk 3]
```diff
@@ -226,14 +226,14 @@ Update your *package.json* like this:
...
   4. Add changes caused by autofixing and reformatting to your commit.
 
-* Every time you commit an `.scss` file:
+* Every time you commit an *.scss* file:
```
Comment (bebraw) [inline manuscript/code-quality/01-linting.md]: I wonder if it's possible to lint file extension formatting somehow. If it was, we wouldn't have to worry about these.

File: manuscript/code-quality/01-linting.md (hunk 1 above)
Comment (sapegin) [reply manuscript/code-quality/01-linting.md]: A link would work here, yeah.

File: manuscript/code-quality/01-linting.md (hunk 2 above)
Comment (sapegin) [reply manuscript/code-quality/01-linting.md]: 🍕

File: manuscript/code-quality/01-linting.md (hunk 3 above)
Comment (sapegin) [reply manuscript/code-quality/01-linting.md]: Nope ;-( No way to tell which is a file name: console.log or package.json.

**Comment by sapegin** [comment]
All fixed!

**Pull request merged**